inference-engine/src/legacy_api/src/ie_layer_validators.cpp

```diff
@@ -449,6 +449,8 @@ void EltwiseValidator::parseParams(CNNLayer* layer) {
         casted->_operation = EltwiseLayer::Pow;
     } else if (op == "mean") {
         casted->_operation = EltwiseLayer::Mean;
+    } else if (op == "abs") {
```

Comment:
I didn't see any changes in the `convert_function_to_cnn_network`. Looks like the conversion won't create Eltwise operation for `Abs` layer.

We always create `CNNLayer` for `Abs` in legacy representation.

Reply:
Sure. Thanks.
Removed abs from here